commands/get/resources/downloader_test.go

```diff
@@ -7,27 +7,29 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/opencontrol/compliance-masonry/tools/vcs/mocks"
 	"errors"
-	"github.com/opencontrol/compliance-masonry/lib/common"
+	commonMocks "github.com/opencontrol/compliance-masonry/lib/common/mocks"
 )
 
 var _ = Describe("Downloader", func() {
-
 	Describe("Constructing a new VCEntrySDownloader", func() {
 		It("should return a downloader of type VCSEntryDownloader", func() {
 			downloader := NewVCSDownloader()
 			assert.IsType(GinkgoT(), vcsEntryDownloader{}, downloader)
 		})
 	})
 	Describe("Downloading Entry from VCS", func(){
-		table.DescribeTable("DownloadEntry", func(e common.Entry, err error) {
+		table.DescribeTable("DownloadRepo", func(err error) {
+			remoteSource := new(commonMocks.RemoteSource)
+			remoteSource.On("GetURL").Return("")
+			remoteSource.On("GetRevision").Return("")
```

Comment:
these don't matter as much since it's not actually downloading something. but i can put something here so people know what to expect!